nengo/params.py

```diff
@@ -416,7 +417,7 @@ def validate(self, instance, function_info):
         super(FunctionParam, self).validate(instance, function)
 
 
-class FrozenObject(object):
+class FrozenObject(with_metaclass(DocstringInheritor)):
```

Comment:
All 3 possibilities (`with_metaclass(DocstringInheritor), FrozenObject`; `FrozenObject, with_metaclass(DocstringInheritor)`; `with_metaclass(DocstringInheritor, FrozenObject)`) passed unit tests for me, but @jgosmann's is undoubtedly the better solution (results in a shorter `mro`, I think), so I pushed a fixup commit with that.